flake.nix

```diff
@@ -25,34 +24,36 @@
 
       stdenvs = [ "gccStdenv" "clangStdenv" "clang11Stdenv" "stdenv" "libcxxStdenv" "ccacheStdenv" ];
 
-      forAllSystems = f: nixpkgs.lib.genAttrs systems (system: f system);
-      forAllSystemsAndStdenvs = f: forAllSystems (system:
+      forAllSystems = nixpkgs.lib.genAttrs systems;
+
+      forAllCrossSystems = nixpkgs.lib.genAttrs crossSystems;
+
+      forAllStdenvs = prefix: f:
```

Comment:
@zimbatm and I discussed using `lib.mapAttrs' where a non-`""` prefix is needed instead of adding this argument.

Reply:
fixed!